srcpkgs/gnome-screensaver/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'gnome-screensaver'
```

Comment:
srcpkgs/gnome-screensaver/template:10: short_desc should not start with an article

Reply:
Should we change the `short_desc` variable into something meaningful? "A new screensaver that can replace xscreensaver" is pretty ironic if you ask me, considering `gnome-screensaver` went archived a long, LONG time ago. 

before xtraeme removed the package in 560c836, the short_desc was "GNOME screen saver and locker". How about adding a "Legacy" before it?

Reply:
Also, does `gnome-screensaver` work for you? I still can't use it gnome-flashback. Nothing happens when locking the screen or suspending.

Reply:
I missed this comment in my emails :( , thanks for the mention.

I haven't had tested `gnome-screensaver` before, can confirm now that it doesn't work. Seems like `gnome-flashback` no longer uses `gnome-screensaver` https://bugzilla.redhat.com/show_bug.cgi?id=1840146.
The inbuilt screensaver [doesn't work](https://gitlab.gnome.org/GNOME/gnome-flashback/-/commit/fd860950be) without a `PAM` file, upstream says it's upto distributions to provide it( Arch,Fedora do and maybe others too).

So maybe we should provide the PAM file too?

Regarding `gnome-screensaver` since it's no longer needed and unmaintained upstream, maybe it should be removed ? CC: package maintainer @shizonic

Reply:
Sorry for the late reply, I messed up my local repository :disappointed: 
Tried this [PAM config](https://git.archlinux.org/svntogit/community.git/tree/trunk/gnome-flashback.pam?h=packages/gnome-flashback) from Arch and it works perfectly ! **(and without `gnome-screensaver`)**

So, `gnome-screensaver` can be safely removed, as no other packages depend on it anymore. I'll be waiting for shizonic's call.

Reply:
>Regarding gnome-screensaver since it's no longer needed and unmaintained upstream, maybe it should be removed ?

I just realized I use it with budgie lol. It's the only locker that does work with it ootb. Maybe we should leave `gnome-screensaver` until it fails to build from source or something...